src/pages/workspaces/WorkspacesList/NoWorkspacesMessage.ts

```diff
@@ -0,0 +1,33 @@
+import _ from 'lodash/fp';
+import { Fragment, ReactNode, useContext } from 'react';
+import { div, h } from 'react-hyperscript-helpers';
+import { NoWorkspacesMessage } from 'src/components/workspace-utils';
+import { cond } from 'src/libs/utils';
+import { CatagorizedWorkspaces } from 'src/pages/workspaces/WorkspacesList/CatagorizedWorkspaces';
+import { WorkspaceFilterValues } from 'src/pages/workspaces/WorkspacesList/WorkspaceFilters';
+import { WorkspaceUserActionsContext } from 'src/pages/workspaces/WorkspacesList/WorkspaceUserActions';
+
+interface NoContentMessageProps {
+  loadingWorkspaces: boolean;
+  loadingSubmissionStats: boolean;
+  workspaces: CatagorizedWorkspaces;
+  filters: WorkspaceFilterValues;
+}
+
+export const NoContentMessage = (props: NoContentMessageProps): ReactNode => {
+  const { loadingWorkspaces, loadingSubmissionStats, workspaces, filters } = props;
+  const { setUserActions } = useContext(WorkspaceUserActionsContext);
+
+  return cond(
+    [loadingWorkspaces, () => h(Fragment, ['Loading...'])],
```

Comment:
Since React Hyperscript Helpers types have been improved to support components that return ReactNode instead of ReactElement, cases like this can return the string directly instead of wrapping in a Fragment.

```suggestion
    [loadingWorkspaces, () => 'Loading...'],
```

In that case, `cond` may need some help figuring out the return type, so adding a type parameter (`cond<ReactNode>(...)`) may be necessary.